src/client/java/minicraft/core/Game.java

```diff
@@ -87,6 +88,7 @@ public static void main(String[] args) {
 		Analytics.GameStartup.ping();
 
 		input = new InputHandler(Renderer.canvas);
+		Renderer.spriteLinker = new SpriteLinker();
```

Comment:
Perhaps this should be in `Renderer.initScreen` instead?

Reply:
It must be created before the creations of tiles and items.

Reply:
Put it before tiles and items are created then.

Reply:
Resource packs also require linker to register sheets.

Reply:
Put it in the constructor of Renderer then.

Reply:
Ok.